radis/lbl/calc.py

```diff
@@ -949,28 +956,22 @@ def _is_at_equilibrium():
     # ------------------
     # Use the standard eq_spectrum / non_eq_spectrum functions
     if _equilibrium:
-        if mode == "cpu":
-            s = sf.eq_spectrum(
-                Tgas=Tgas,
-                mole_fraction=mole_fraction,
-                path_length=path_length,
-                name=name,
-            )
-
-        elif mode in ("gpu", "emulated_gpu"):
-            s = sf.eq_spectrum_gpu(
-                Tgas=Tgas,
-                mole_fraction=mole_fraction,
-                pressure=pressure,
-                path_length=path_length,
-                name=name,
-                backend=("cpu-cuda" if mode == "emulated_gpu" else "gpu-cuda")
-                # emulate=(True if mode == "emulated_gpu" else False),
-            )
-        else:
+        
+        if mode not in ['cpu', 'gpu']:
             raise ValueError(
-                f"mode= should be one of 'cpu', 'gpu', 'emulated_gpu' (GPU code running on CPU). Got {mode}"
+                f"mode= should be 'cpu' or 'gpu'; got {mode}"
             )
+            return
+        
+        eq_spectrum_mode = sf.eq_spectrum if mode == 'cpu' else sf.eq_spectrum_gpu
+        
+        s = eq_spectrum_mode(
+            Tgas=Tgas,
+            mole_fraction=mole_fraction,
+            path_length=path_length,
```

Comment:
pressure is missing (note : it as missing before already !?)

Reply:
Good point, same for non-eq it seems..
I updated it

Reply:
@dcmvdbekerom

Reply:
It's updated in the development branch: https://github.com/radis/radis/blob/7350e1702cd02b2cde0e993163335e3363bafa7a/radis/lbl/calc.py#L968